testng-core/src/test/java/test/listeners/issue2880/IssueTest.java

```diff
@@ -0,0 +1,74 @@
+package test.listeners.issue2880;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.testng.TestNG;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlSuite;
+import test.SimpleBaseTest;
+
+public class IssueTest extends SimpleBaseTest {
```

Comment:
Please move this test method into `test.listeners.ListenersTest`. we are trying to ensure that all the tests reside in the same class. The test classes that this test uses can reside in its own package (`issue2880` in this case)

Reply:
Accepted and Done.